federated-client-authentication/kubernetes/keycloak.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: keycloak
+  labels:
+    app: keycloak
+spec:
+  ports:
+    - name: http
+      port: 8080
+      targetPort: 8080
+  selector:
+    app: keycloak
+  type: LoadBalancer
```

Comment:
Do we need a Load Balancer here? Apparently we're using an Ingress, so this seems redundant.